pandas/tests/frame/test_timezones.py

```diff
@@ -196,3 +196,15 @@ def test_tz_localize_convert_copy_inplace_mutate(self, copy, method, tz):
                              index=date_range('20131027', periods=5,
                                               freq='1H', tz=tz))
         tm.assert_frame_equal(result, expected)
+
```

Comment:
these should go with the appropriate tests, I think in test_nanops or maybe tests/reductions)

Reply:
Sure. Moved to `tests/reductions`